examples/rbd/storageclass.yaml

```diff
@@ -48,6 +52,25 @@ parameters:
    # a unique ID matching KMS ConfigMap. The ID is only used for correlation to
    # config map entry.
    # encryptionKMSID: <kms-config-id>
+
+   # Add topology constrained pools configuration, if topology based pools
+   # are setup, and topology constrained provisioning is required.
+   # For further information read TODO<doc>
+   # topologyConstrainedPools: |
+   #   [{"poolName":"pool0",
```

Comment:
This will start to get ugly, because in the topology constrained pools section we will now add one more parameter.

I forget what the limitation is to directly create the image OMap and the image on an EC pool, or is it just not supported? Tagging @dillaman as well for details.

If it is an RBD behavior, then we have no go but to add it to this section as well. But, it gets trickier than just that, we now need 2 pools per topology domain, one replicated for the OMap and image, and one EC pool for the data.

Reply:
Need clarification on this one.